packages/firebase/index.d.ts

```diff
@@ -5277,6 +5277,103 @@ declare namespace firebase.User {
   }
 }
 
+/**
+ * Developer-facing Firebase Auth error codes.
+ */
+declare namespace firebase.auth.ErrorCode {
+  var ADMIN_ONLY_OPERATION: string;
```

Comment:
Shall we use the enum value as type instead of `string`?

Reply:
Done.